src/pages-and-resources/discussions/data/redux.test.js

```diff
@@ -350,9 +372,12 @@ describe('Data layer integration tests', () => {
           // but we technically send it to the thunk, so here it is.
           divideByCohorts: true,
           allowDivisionByUnit: true,
-          divideCourseWideTopics: true,
-          divideGeneralTopic: true,
-          divideQuestionsForTAsTopic: true,
+          divideCourseTopics: false,
+          dividedDiscussionsIds,
+          discussionTopics: [
+            { name: 'Edx', id: '13f106c6-6735-4e84-b097-0456cff55960' },
+            { name: 'General', id: 'course-generated-id-123-client-made-this-up' },
```

Comment:
Can we use id: "default" for General topic, which is the expected response from the API for the general topic id

Reply:
Sure